pkg/controller/atlasdeployment/serverless_private_endpoint.go

```diff
@@ -0,0 +1,356 @@
+package atlasdeployment
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/util/stringutil"
+
+	"go.uber.org/zap"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1/status"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+
+	mdbv1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1/provider"
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/controller/workflow"
+)
+
+const (
+	SPEStatusDeleting = "DELETING"
+
+	SPEStatusReserved             = "RESERVED"              // 1
```

Comment:
Maybe change comments to something like "stage 1", "stage 2" so that one won't need to guess the meaning?

Reply:
Done